src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figuresuppliers.FigureSupplier;
+
+public class Main {
+    private static final int SIZE = 4;
+
+    public static void main(String[] args) {
+        Figure[] arr = new Figure[SIZE];

```

Comment:
Please try to choose more descriptive variable names. arr is too generic, code is not always read from top to bottom)) There might be a case when some developer opens your code on the 13th line and wants to understand what you do there, but in this case, he would need to go all the way up to figure out what is stored in arr variable. A better naming choice would be "figures" or something similar